frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts

```diff
@@ -46,7 +48,7 @@ export const getDefaultFieldSettings = (
   title: "",
   description: "",
   placeholder: "",
-  order: 0,
+  order: 100,
```

Comment:
Why `100` is a default?

Reply:
By default we want to add to the end of the list. I think we're unlikely to have forms with 100 parameters.

Reply:
I'm worried it can be a bit obvious. The function expects an array of `parameters` as an argument; can we set the default order to be `parameters.length` instead? Maybe add a comment if there's no clean way to get the last index?